lib/rubocop/ast/node/mixin/transmitter_node.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module AST
+    # Common predicates for nodes that have receivers:
+    # `send`, `csend`, `def`, `defs`, `super`, `zsuper`
+    #
+    # @note this mixins expects `#receiver` to be implemented
+    module TransmitterNode
```

Comment:
Yeah. Couldn't come up with anything better for a node that has a receiver at the point. 🤔